src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -169,8 +175,13 @@ private String getAuthorizationHeader() {
 
     private Request getService(final String url, boolean isHttpGet) {
         String contentType = "application/" + ((isEncrypted) ? "jose+json" : "json");
-        Request request = new Request(url)
-                .addHeader("Authorization", getAuthorizationHeader())
+        Request request;
```

Comment:
Minor: May be cleaner to use ternary operator here
```
Request request = usesProxy() ? new Request(url, proxy) : new Request(url);
```

Reply:
I do think this is a cleaner change, will impl